NEWS.md

```diff
@@ -1,5 +1,10 @@
 # dtplyr (development version)
 
+* `left_join()`, `right_join()`, and `inner_join()` are now always translated to
```

Comment:
Comment on why this is a good thing?

Reply:
I'm not sure there are direct advantages for the user. My main motivation was to fix handling of joining by differently named columns and handling of duplicate column names. Code wise I think it is a bit nicer that it isn't necessary anymore to distinguish between "simple" and "non-simple" joins.

Reply:
Ok, makes sense.

Reply:
Could you please include a bit of this justification in the news bullet? i.e. it makes the translation a bit longer for simple merges, but the advantage is that ...